src/lib.rs

```diff
@@ -913,8 +913,8 @@ mod tests {
             assert_eq!(None, iter.next());
         }
 
-        assert_eq!(17, map["a"]);
-        assert_eq!(23, map["b"]);
+        assert_eq!(17, map[&"a"]);
+        assert_eq!(23, map[&"b"]);
```

Comment:
`Index` and `get` use `Borrow`, and `&str: Borrow<str>`. (For what it's worth, the crate type-checks with just `map["a"]`, but that bug prevents it from being built.)